packages/ui-color-picker/src/ColorMixer/__tests__/ColorMixer.test.tsx

```diff
@@ -128,4 +146,1591 @@ describe('<ColorMixer />', () => {
+        await mount(
+          <ColorMixer
+            {...testInputLabels}
+            {...testScreenReaderLabels}
+            onChange={onChange}
+          />
+        )
+        const component = await ColorMixerLocator.find()
+        const hueSlider = await component.findColorSlider()
+        const rect = hueSlider.getBoundingClientRect()
+        await hueSlider.mouseDown({ clientX: rect.x + rect.width / 2 })
+        expect(onChange).to.have.been.called()
+      })
+      it('click at the end of the slider', async () => {
+        const onChange = stub()
+        await mount(
+          <ColorMixer
+            {...testInputLabels}
+            {...testScreenReaderLabels}
+            onChange={onChange}
+          />
+        )
+        const component = await ColorMixerLocator.find()
+        const hueSlider = await component.findColorSlider()
+        const rect = hueSlider.getBoundingClientRect()
+        await hueSlider.mouseDown({ clientX: rect.x + rect.width })
+        expect(onChange).to.have.been.called()
+      })
+
+      it('click at the begin of the slider', async () => {
+        const onChange = stub()
+        await mount(
+          <ColorMixer
+            value="#00FF00"
+            {...testInputLabels}
+            {...testScreenReaderLabels}
+            onChange={onChange}
+          />
+        )
+        const component = await ColorMixerLocator.find()
+        const hueSlider = await component.findColorSlider()
+        const rect = hueSlider.getBoundingClientRect()
+        await hueSlider.mouseDown({ clientX: rect.x })
+        expect(onChange).to.have.been.calledTwice()
+      })
+
+      it('click at the outside of the slider to the left', async () => {
+        const onChange = stub()
+        await mount(
+          <ColorMixer
+            value="#00FF00"
+            {...testInputLabels}
+            {...testScreenReaderLabels}
+            onChange={onChange}
+          />
+        )
+        const component = await ColorMixerLocator.find()
+        const hueSlider = await component.findColorSlider()
+        const rect = hueSlider.getBoundingClientRect()
+        await hueSlider.mouseDown({ clientX: rect.x - 100 })
+        expect(onChange).to.have.been.calledTwice()
+      })
+
+      it('click at the outside of the slider to the right', async () => {
+        const onChange = stub()
+        await mount(
+          <ColorMixer
+            value="#00FF00"
+            {...testInputLabels}
+            {...testScreenReaderLabels}
+            onChange={onChange}
+          />
+        )
+        const component = await ColorMixerLocator.find()
+        const hueSlider = await component.findColorSlider()
+        const rect = hueSlider.getBoundingClientRect()
+        await hueSlider.mouseDown({ clientX: rect.x + rect.width + 100 })
+        expect(onChange).to.have.been.calledTwice()
+      })
+    })
+
+    it("hue value change with 'a' key pressed", async () => {
```

Comment:
I'd put these the "hue value change with 'xyz' key pressed" tests to a single test with a for loop